sql/src/main/java/io/crate/operation/projectors/SortingTopNProjector.java

```diff
@@ -126,13 +126,6 @@ public void fail(Throwable t) {
     }
 
     @Override
-    public void repeat() {
-        final int resultSize = Math.max(pq.size() - offset, 0);
```

Comment:
I think this was only used for the resultSize == 0 case, because otherwise the rowEmitter calls finish and manages the repeatable.
Currently repeat is never used for empty row cases, but I should probably add it back anyway?